src/library/Utilities/RemoteContainerConnectionDetailsResolver.cs

```diff
@@ -328,27 +343,34 @@ internal class RemoteContainerConnectionDetailsResolver
                 {
                     throw new UserVisibleException(_operationContext, Resources.ResourceNotSupportedFormat, Product.Name);
                 }
+                _log.Info("OwningSets name:" + owningSets);

```

Comment:
I will remove these temporary log statements which was added to triage once user confirms it is working. Thanks.